frontend/packages/helm-plugin/src/topology/components/HelmReleaseGroup.tsx

```diff
@@ -53,10 +53,10 @@ const HelmReleaseGroup: React.FC<HelmReleaseGroupProps> = ({
   dragging,
   dragNodeRef,
 }) => {
-  const [hover, hoverRef] = useHover();
-  const [innerHover, innerHoverRef] = useHover();
+  const [hover, hoverRef] = useHover<SVGGElement>();
+  const [innerHover, innerHoverRef] = useHover<Element>();
   const [{ dragging: labelDragging }, dragLabelRef] = useDragNode(noRegroupDragSourceSpec);
-  const nodeRefs = useCombineRefs(innerHoverRef, dragNodeRef);
+  const nodeRefs = useCombineRefs(innerHoverRef, dragNodeRef as React.Ref<Element>);
```

Comment:
does this work to remove the type assertion in line 90?

```suggestion
  const nodeRefs = useCombineRefs<SVGGElement>(innerHoverRef, dragNodeRef);
```

Reply:
Yeah removing it in line 90 isn't give me errors